filestore/s3_test.go

```diff
@@ -30,6 +32,17 @@ type TestSuite struct {
 	deleteTempDir bool
 }
 
+func httpClient() *http.Client {
+
+	customTransport := &http.Transport{
+	    TLSClientConfig: &tls.Config{InsecureSkipVerify: false},
+        }
+	customHTTPClient := &http.Client{
+                Transport:        customTransport,
```

Comment:
indent funkiness here

Reply:
IIRC compiling auto formats the code - if not maybe the best thing to do is just run the auto formatter that comes with go.